beacon_node/store/src/hot_cold_store.rs

```diff
@@ -1054,9 +1065,12 @@ impl<E: EthSpec, Hot: ItemStore<E>, Cold: ItemStore<E>> HotColdDB<E, Hot, Cold>
                     }
                     true
                 }
-                StoreOp::DeleteDataColumns(block_root) => {
+                StoreOp::DeleteDataColumns(block_root, _column_indices) => {
```

Comment:
Good question. I am not able to justify this log. I went ahead an implemented pruning to see how it would look like. Please see https://github.com/sigp/lighthouse/pull/5890/commits/7d72c364eb7142eef1ca20b3f5fe4d261bf67d2e